src/time/Storage.cpp

```diff
@@ -242,14 +242,9 @@ int Storage::computeUsedDegree(int requestedDegree, int numberOfAvailableSamples
   return std::min(requestedDegree, numberOfAvailableSamples - 1);
 }
 
-time::Sample Storage::getSampleAtBeginning()
+Stample Storage::getStampleAtEnd()
```

Comment:
To avoid the copies.

```suggestion
Stample& Storage::getStampleAtEnd()
```

Reply:
Done [1bf2cb7](https://github.com/precice/precice/pull/2203/commits/1bf2cb7083375b4ba6f318d81e6f08bd9872a505)